compile/README.md

```diff
@@ -26,11 +38,34 @@ NOTE: The files are not statically compiled, and hence any pypi dependencies wil
     ```
 2. **Variables**
     - `REMOVE_BRANCHES: "true"` removes all destination branches except main before pushing
-    - `GIT_STRATEGY: clone` !Important as it makes sure that source repo doesn't contain cached files
-    - `GIT_DEPTH: 1` can be set accordingly
+    - (Required) `GIT_STRATEGY: clone` !Important as it makes sure that source repo doesn't contain cached files
```

Comment:
If this variable is definitely required, should it also be included in the reference implementation (Line 9 of your readme) ?